src/data/showcases.js

```diff
@@ -1171,6 +1171,15 @@ export const Showcases = [
     source: null,
     tags: ["analytics", "pooltool"],
   },
+  {
+    title: "Summon Platform",
+    description:
+      "A DAO creation and governance platform on the Cardano blockchain.",
+    preview: require("./showcase/summonplatform.png"),
+    website: "https://app.summonplatform.io/",
```

Comment:
```suggestion
    website: "https://summonplatform.io/",
```
What do you think about landing on the home page, rather than the app page which is pretty bare (except for a link to the medium blog) for new users?  First time visitors I think will need the introduction on the home page.

Reply:
Thank you for bringing this up @rphair . I was considering this issue when I created the PR and decided to go for the link to the application as I thought it was an unwritten rule to link directly to the application and not the home page of the project, similar to [Minswap](https://developers.cardano.org/showcase?tags=dex&name=minswa) or [AADA](https://developers.cardano.org/showcase/?name=aada) (which we also need to change as the rebranded to Lenfi).

I agree that first time visitors would benefit from landing on https://summonplatform.io/

I'm happy to go along with this change.